bin/play.py

```diff
@@ -28,7 +28,8 @@ def play_sample(samples, index):
         sys.exit(1)
     sample = samples[index]
     print('Sample "{}"'.format(sample.sample_id))
-    print('  "{}"'.format(sample.transcript))
+    if isinstance(sample, LabeledSample):
+        print('  "{}"'.format(sample.transcript))
```

Comment:
This is not very Pythonic. I suggest adding `__str__` methods to the class hierarchy so that this can just be a `print(sample)`.

Reply:
Generally agree, but this formatting is application specific (printing separate lines for the ID and the transcript for better readability).